en/mozorg/about/governance/policies/participation.ftl

```diff
@@ -0,0 +1,107 @@
+participation-unwelcome-sexual-attention = Unwelcome Sexual Attention or Physical Contact
+participation-unwelcome-sexual-attention-or = Unwelcome sexual attention or unwelcome physical contact is not acceptable. This includes sexualized comments, jokes or imagery in interactions, communications or presentation materials, as well as inappropriate touching, groping, or sexual advances. This includes touching a person without permission, including sensitive areas such as their hair, pregnant stomach, mobility device (wheelchair, scooter, etc) or tattoos. This also includes physically blocking or intimidating another person. Physical contact or simulated physical contact (such as emojis like “kiss”) without affirmative consent is not acceptable. This includes sharing or distribution of sexualized images or text.
+participation-disruptive-behavior = Disruptive Behavior
+participation-sustained-disruption = Sustained disruption of events, forums, or meetings, including talks and presentations, will not be tolerated. This includes:
+participation-talking-over-or-heckling = ‘Talking over’ or ‘heckling’ speakers.
+participation-drinking-alcohol-to = Drinking alcohol to excess or using recreational drugs to excess, or pushing others to do so.
+participation-making-derogatory = Making derogatory comments about those who abstain from alcohol or other substances, pushing people to drink, talking about their abstinence or preferences to others, or pressuring them to drink - physically or through jeering.
+participation-otherwise-influencing = Otherwise influencing crowd actions that cause hostility in the session.
+participation-influencing-unacceptable = Influencing Unacceptable Behavior
+participation-we-will-treat-influencing = We will treat influencing or leading such activities the same way we treat the activities themselves, and thus the same consequences apply.
+participation-consequences-of-unacceptable = Consequences of Unacceptable Behavior
+participation-bad-behavior-from = Bad behavior from any Mozillian, including those with decision-making authority, will not be tolerated. Intentional efforts to exclude people (except as part of a consequence of the guidelines or other official action) from { -brand-name-mozilla } activities are not acceptable and will be dealt with appropriately.
+participation-reports-of-harassmentdiscrimination = Reports of harassment/discrimination will be promptly and thoroughly investigated by the people responsible for the safety of the space, event or activity. Appropriate measures will be taken to address the situation.
+participation-anyone-asked-to-stop = Anyone asked to stop unacceptable behavior is expected to comply immediately. Violation of these guidelines can result in you being ask to leave an event or online space, either temporarily or for the duration of the event, or being banned from participation in spaces, or future events and activities in perpetuity.
```

Comment:
- `... being ask ...` should be `... being asked ...`
- `... you being ask ...` - `you` here seems strange. It started as `Anyone asked to stop...`. Need to use a different pronoun.